app/views/assignments/_age_request_select.html.slim

```diff
@@ -0,0 +1,11 @@
+- age_request_filter = params[:q] && params[:q][:age_request_cont].present? ? params[:q][:age_request_cont] : 'all'
```

Comment:
Complicated logic like this should be moved into a helper, also allows easier testing.